quint/testFixture/_1024importFromSyntaxError.json

```diff
@@ -1 +1 @@
-{"stage":"parsing","warnings":[],"modules":[{"id":10,"name":"I","declarations":[{"id":9,"kind":"def","name":"x","qualifier":"val","expr":{"id":8,"kind":"bool","value":true}}]},{"id":3,"name":"importer","declarations":[{"id":2,"kind":"import","defName":"*","protoName":"I","fromSource":"./_1025importeeWithError"}]}],"table":{},"errors":[{"explanation":"[QNT000] mismatched input ''' expecting STRING","locs":[{"source":"mocked_path/testFixture/_1024importFromSyntaxError.qnt","start":{"line":1,"col":18,"index":36},"end":{"line":1,"col":18,"index":36}}]}]}
\ No newline at end of file
+{"stage":"parsing","warnings":[],"modules":[{"id":10,"name":"I","declarations":[{"id":9,"kind":"def","name":"x","qualifier":"val","expr":{"id":8,"kind":"let","opdef":{"id":6,"kind":"def","name":"y","qualifier":"val","expr":{"id":5,"kind":"bool","value":true}},"expr":{"id":7,"kind":"bool","value":true}}}]},{"id":3,"name":"importer","declarations":[{"id":2,"kind":"import","defName":"*","protoName":"I","fromSource":"./_1025importeeWithError"}]}],"table":{},"errors":[{"explanation":"[QNT000] mismatched input ''' expecting STRING","locs":[{"source":"mocked_path/testFixture/_1024importFromSyntaxError.qnt","start":{"line":1,"col":18,"index":36},"end":{"line":1,"col":18,"index":36}}]}]}
```

Comment:
Instead of a simple undefined value, we now have something that resembles the original structure a bit more, since each stack has it's own default value.

Instead of `val x = true` we have `val x = val y = true { true }`